docs/setup/settings.asciidoc

```diff
@@ -82,52 +83,59 @@ Elasticsearch nodes on startup.
-prove Kibana's identity when it makes an outbound request to your Elasticsearch cluster. If the file contains any additional certificates,
-those will be used as a trusted certificate chain for your Elasticsearch cluster. This chain is used to establish trust when Kibana creates
-an SSL connection with your Elasticsearch cluster. In addition to this setting, trusted certificates may be specified via
-`elasticsearch.ssl.certificateAuthorities` and/or `elasticsearch.ssl.truststore.path`.
+`elasticsearch.ssl.keystore.path:`:: Path to a PKCS#12 keystore that contains an X.509 client certificate and its corresponding private key.
+These are used by {kib} to authenticate itself when making outbound SSL/TLS connections to {es}. For this setting to take effect, the
+`xpack.security.http.ssl.client_authentication` setting in {es} must also be set to `"required"` or `"optional"` to request a client
+certificate from {kib}.
 +
 --
+If the keystore contains any additional certificates, those will be used as a trusted certificate chain for {es}. This chain is used by
+{kib} to establish trust when making outbound SSL/TLS connections to {es}. In addition to this setting, trusted certificates may be
+specified via `elasticsearch.ssl.certificateAuthorities` and/or `elasticsearch.ssl.truststore.path`.
+
 NOTE: This setting cannot be used in conjunction with `elasticsearch.ssl.certificate` or `elasticsearch.ssl.key`.
 --
 
-`elasticsearch.ssl.keystore.password:`:: The password that will be used to decrypt the key store and its private key. If your key store has
-no password, leave this unset. If your key store has an empty password, set this to `""`.
+`elasticsearch.ssl.keystore.password:`:: The password that will be used to decrypt the keystore that is specified via
+`elasticsearch.ssl.keystore.path`. If the keystore has no password, leave this unset. If the keystore has an empty password, set this to
+`""`.
 
-`elasticsearch.ssl.truststore.path:`:: Path to a PKCS #12 trust store that contains one or more X.509 certificates. This may consist of a
-root certificate authority (CA) and one or more intermediate CAs, which make up a trusted certificate chain for your Elasticsearch cluster.
-This chain is used to establish trust when Kibana creates an SSL connection with your Elasticsearch cluster. In addition to this setting,
-trusted certificates may be specified via `elasticsearch.ssl.certificateAuthorities` and/or `elasticsearch.ssl.keystore.path`.
+`elasticsearch.ssl.truststore.path:`:: Path to a PKCS#12 trust store that contains one or more X.509 certificate authority (CA) certificates
+which make up a trusted certificate chain for {es}. This chain is used by {kib} to establish trust when making outbound SSL/TLS connections
+to {es}.
++
+In addition to this setting, trusted certificates may be specified via `elasticsearch.ssl.certificateAuthorities` and/or
```

Comment:
Per earlier comment, it would be nice to clarify why you'd use one or the other of these truststore/keystore/CA settings.

Reply:
Copy-pasting my other response:
Yes, you can use all three at the same time if you wanted. It's a unlikely scenario but it is possible that you may have some CA certificates in PEM format and some other ones in a PKCS#12 trust store. It was a lot simpler logic to combine than all than to try to throw an error if CA certs were detected in multiple places.